### PR TITLE
Add page for Dongze Sun

### PR DESCRIPTION
**Pull request by dongzesun**
Add page for Dongze Sun

### CODE REVIEW

File: _people/Dongze_Sun.markdown [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+advisor: Saul A. Teukolsky
+phone:
+email: dzsun@caltech.edu
+address: 360 Cahill
+specialties: Numerical Relativity, binary black holes, waveforms, post-Newtonian
+---
+
+##### Research Projects
+
+I focus on various aspects of binary black holes, including waveform hybridization, boundary conditions for numerical relativity, post-newtonian calculations, and numerical algrithims.
```
Comment (duetosymmetry) [inline _people/Dongze_Sun.markdown]: Typo, algrithims -> algorithms

Comment (dongzesun) [reply _people/Dongze_Sun.markdown]: Sorry! Now it's fixed

**Comment by duetosymmetry** [comment]
Please squash

**Comment by dongzesun** [comment]
Squashed

**Pull request merged**